Copyedit/revise English osu! livestreaming guide (#7715)
ppy/osu-wiki: wiki/Guides/Livestreaming_osu!/en.md

![](img/dashboard.png "Entering the Creator Dashboard")

> In the Creator Dashboard, you can configure stream information, chat moderation, and profile info, among other customizations. Most important to getting started is configuring your stream's information. From the main page of the dashboard, click on the `Stream Manager` button on the left, and then choose the `Edit Stream Info` option on the right.

Reviewer: british english

```suggestion
In the Creator Dashboard, you can configure stream information, chat moderation, and profile info, among other customisations. Most important to getting started is configuring the stream information. From the main page of the dashboard, click on the `Stream Manager` button on the left, and then choose the `Edit Stream Info` option on the right.
```

Reviewer 2: _In the Creator Dashboard, you can configure stream information, chat moderation, and profile info, among other customizations._

If you are using a comma I think "and" isn't necessary?

Reviewer: to me it reads better

Author: Thanks, added.

![](img/configuration-wizard.png "OBS Studio's auto-configuration wizard")

> The next section to configure contains video settings. Choose the resolution of your screen for the `Base (Canvas) Resolution` field (should be detected automatically), and `Either 60 or 30, but prefer 60 when possible` for the `FPS` field.

Reviewer: ```suggestion
The next step is to configure some video settings. Choose the resolution of your screen for the `Base (Canvas) Resolution` field (should be detected automatically), and `Either 60 or 30, but prefer 60 when possible` for the `FPS` field.
```

Author: Subjective, I tried to keep it closer to the original, but your suggestion is better.

Reviewer: it doesn't have to be "close to the original" - if you rewrote the entire article but better that'd be fine too

## What's next?

> Now, after you configure OBS Studio and your Twitch account, click `Start Streaming` in OBS Studio to make a livestream. Your stream will appear on [Twitch](https://www.twitch.tv/directory/game/osu!) and [osu! website's livestream listing](https://osu.ppy.sh/community/livestreams) in a couple of minutes. That's all you need to know to start your journey with streaming. Of course, this guide only covers the basics of livestreaming; there are many ways to go beyond and improve your livestream. So, some important points to consider:

Reviewer 2: ```suggestion
Now, after configuring OBS Studio and your Twitch account, click `Start Streaming` in OBS Studio to make a livestream. Your stream will appear on [Twitch](https://www.twitch.tv/directory/game/osu!) and [osu! website's livestream listing](https://osu.ppy.sh/community/livestreams) in a couple of minutes. That's all you need to know to start your journey with streaming. Of course, this guide only covers the basics of livestreaming; there are many ways to go beyond and improve your livestream. So, some important points to consider:
```

Author: Good idea.
